core/loader.rb

```diff
@@ -7,6 +7,12 @@
 # @note Include here all the gems we are using
 require 'rubygems'
 require 'bundler/setup'
+
+# For some reason, on Ruby 2.5+, msgpack needs to be loaded first,
+# else metasploit integration dies due to undefined `to_msgpack`.
+# Works fine on Ruby 2.4
+require 'msgpack'
```

Comment:
Just a thought here, `msgpack` is not listed as a dependency in the beef `Gemfile`.  I am not super familiar with how the interaction of dependencies are being processed however adding the dependency directly might produce a more universal revision.